src/sasmita/DivisiableBySevenandThirteen.java

```diff
@@ -0,0 +1,19 @@
+package sasmita;
+
+public class DivisiableBySevenandThirteen{
+	
+	void divisiableNumbers(){
+		for(int index=40; index>=5; index--) {
```

Comment:
make it generic, so that you can use this code for different ranges

Reply:
sure